tools/clang/test/lit.cfg

```diff
@@ -269,6 +269,41 @@ config.substitutions.append( ('%itanium_abi_triple', makeItaniumABITriple(config
 config.substitutions.append( ('%ms_abi_triple', makeMSABITriple(config.target_triple)) )
 config.substitutions.append( ('%dxc', lit.util.which('dxc', llvm_tools_dir)) )
 
+bin_dir = os.path.join(config.llvm_obj_root, lit_config.params['build_mode'], 'bin')
+config.environment['PATH'] = str.format('{};{}',config.environment['PATH'], bin_dir)
+
+
+# Displaying the parent directory of the script
+cur_dir = os.path.dirname(__file__)
+
+config.substitutions.append( ('%batch',
+                            'dxc_batch' ))
+
+config.substitutions.append( ('%dxv',
+                            'dxv' ))
+
+config.substitutions.append( ('%dxa',
+                            'dxa') )
+
+if platform.system() not in ['Windows']:
```

Comment:
LLVM's lit tests require that the git command line tools be on your path for Windows. It seems reasonable to adopt that requirement for DXC so that we don't need to do things like this.

Reply:
Removed the bat file.